### PR TITLE
Updated Flink Package to 1.3.1.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

**Comment by joerg84** [comment]
@xinfenqiang FYI

**Comment by joerg84** [comment]
Tested and verified on 1.9.

File: repo/packages/F/flink/2/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "packagingVersion": "3.0",
+
+    "name": "flink",
+    "version": "1.3.1-1.0",
+    "description": "Apache Flink® is an open source platform for distributed stream and batch data processing. Further information can be found here: https://github.com/dcos/examples/flink",
```
Comment (ryadav88) [inline repo/packages/F/flink/2/package.json]: should be `https://github.com/dcos/examples/tree/master/flink`

**Comment by joerg84** [comment]
@ryadav88 PTAL

**Pull request merged**